pandas/core/strings.py

```diff
@@ -632,9 +633,9 @@ def str_split(arr, pat=None, n=None, return_type='series'):
     pat : string, default None
         String or regular expression to split on. If None, splits on whitespace
     n : int, default None (all)
-    return_type : {'series', 'index', 'frame'}, default 'series'
-        If frame, returns a DataFrame (elements are strings)
-        If series or index, returns the same type as the original object
+    return_type : {'same', 'expand'}, default 'series'
+        If expand, returns a DataFrame (elements are strings)
```

Comment:
Can you add puntcuation? (so a `.` at the end of the line) -> when converted to html, the line breaks are not preserved as in raw text here.

Also, can you quote expand (like `'expand'`), then it is clearer it is the value you give to the argument.